### ISSUE TITLE
Dropping launch.json "args"

### ISSUE THREAD

**Issue by jtheyer** [ArturoDent/launch-config#6]
Hi ArturoDent, first I want to say this extension is amazing and super helpful! I recently installed it and went over the documentation but am unsure of where to find this issue, if any, as it looks like the "arg" for the keybind(s) is another topic. 
  The problem I'm having is the "args" from launch.json seem to not be getting executed.  
  
![image](https://user-images.githubusercontent.com/23281810/134234962-11858cb8-9f84-4188-b253-94163c8e960b.png)

Basically, the "--outputdir=..." (as you'd expect) redirects result files to the specified location, however, they are going into the top level ${workspaceFolder} when ran via Launches keybind.

**Comment by jtheyer** [comment]
Update: 
Seems to be working ok now. Not really sure what happened, but the result files are going where they're supposed to now. Maybe this can be closed without any need to be concerned?

**Comment by ArturoDent** [comment]
No problem.  All the entries, including `"args"` are just passed through and not touched by the extension.  If  the configuration works when run manually directly, it should work in the extension.
